src/import/saga.ts

```diff
@@ -159,9 +160,11 @@ export function* importBackupPhraseSaga({ phrase, useEmptyWallet }: ImportBackup
       ValoraAnalytics.track(AppEvents.redux_store_recovery_success, { account })
     }
     ValoraAnalytics.track(OnboardingEvents.wallet_import_success)
+    yield call(initializeAccountSaga)
+
+    const numberAlreadyVerifiedCentrally = yield select(numberVerifiedCentrallySelector)
     const skipVerification = yield select(skipVerificationSelector)
-    if (skipVerification) {
-      yield put(initializeAccount())
+    if (skipVerification || numberAlreadyVerifiedCentrally) {
```

Comment:
What do you think of covering this new case with a test too?

Reply:
yup, added!